chaincode/ledger_db.go

```diff
@@ -494,3 +494,13 @@ func (db *LedgerDB) AddTupleEvent(tupleKey string) error {
 	}
 	return nil
 }
+
+// AddComputePlanEvent add the output tuple matching the tupleKey to the event struct
```

Comment:
I think you forgot to update the comment for the compute plan case

Reply:
done